tests/components/group/test_init.py

```diff
@@ -1041,7 +1108,6 @@ async def test_cover_added_after_group(hass):
         "cover.downstairs",
     ]
 
-    assert await async_setup_component(hass, "cover", {})
```

Comment:
Why is it ok to remove the set up of the cover integration?

Reply:
Hi, thanks for pointing this out. I have removed the cover integration by accident....not enough sleep :shrug: . However the test did not fail without it ...

Reply:
Yeah, it's not a good song that the test didn't fail. Not sure exactly about all the logic going on here.

Let's try to keep the tests as similar as possible to the old tests and we can figure this out later.

Reply:
Yes, thanks. I have reverted the tests to the old tests style...